src/App.java

```diff
@@ -1,5 +1,56 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
 public class App {
-    public static void main(String[] args) throws Exception {
-        System.out.println("Hello, World!");
+    /*loads the flashcards from flashcard.txt*/
+    private List<Flashcard> loadFlashcards(String filename) {
+        List<Flashcard> flashcards = new ArrayList<>();/*stores flashcard object*/
+        try {
+            Scanner fileScanner = new Scanner(new File(filename));

```

Comment:
ideally, the file I/O should be stored in a separate file - in my version for madlibs, I put file i/o in the same place as person i/o, the `Printer` class